activities.json

```diff
@@ -0,0 +1,108 @@
+[
+  {
+    "title": "The Smile Trainer",
+    "emoji": "😃",
+    "description": "hold your smile for 10 seconds and uplift your mood",
+    "forEmotion": ["sadness", "anger", "fear"],
+    "tool": true,
+    "link": "/smiletrainer"
+  },
+  {
+    "title": "Inspirational Quotes",
+    "emoji": "🌟📖",
+    "description": "hold your smile for 10 seconds and uplift your mood",
```

Comment:
I already reviewed this 🤔
Why is it in several PRs?

Reply:
😇